packages/@biomejs/plugin-api/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                              Apache License
```

Comment:
Why does this crate have custom licenses? I don't think we need these?

Reply:
`@biomejs/plugin-api` will be shipped on npm as a dedicated package for types, so I copied the licences from `@biomejs/js-api`.